src/ensembl/src/server/helpers/getConfigForClient.ts

```diff
@@ -36,9 +36,10 @@ const getEnvironment = () => {
   };
 };
 
+// TODO: Revert the following change before merging
 const getKeys = () => {
   return {
-    googleAnalyticsKey: process.env.GOOGLE_ANALYTICS_KEY ?? ''
+    googleAnalyticsKey: process.env.GOOGLE_ANALYTICS_KEY ?? 'UA-58710484-17'
```

Comment:
No, we shouldn't hard-code the analytics key in our built javascript.

Reply:
Reverted this change.